opensearch/plugindefinition.yaml

```diff
@@ -53,18 +53,47 @@ spec:
       required: true
       type: list
     - name: cluster.usersCredentials.logs.username
-      description: "Primary OpenSearch user username"
+      description: "Primary OpenSearch username for log ingestion."
       required: true
       type: secret
     - name: cluster.usersCredentials.logs.password
-      description: "Primary OpenSearch user password"
+      description: "Primary OpenSearch user password for log ingestion."
       required: true
       type: secret
     - name: cluster.usersCredentials.logs2.username
-      description: "Secondary OpenSearch user username for failover authentication"
+      description: "Secondary OpenSearch username for failover log ingestion."
       required: true
       type: secret
     - name: cluster.usersCredentials.logs2.password
-      description: "Secondary OpenSearch user password for failover authentication"
+      description: "Secondary OpenSearch user password for failover log ingestion."
```

Comment:
I disagree with these changes, imo "failover for log ingestion" does not make a lot of sense. Failover _is_ authentication related no? Sure the user is technically for log ingestion (so those changes are fine), but not the failover mechanism.

If anything (not saying you should use this because it is quite verbose) it should be closer to this:

```
Secondary user, for log ingestion, password to support failover.
```

Same for username.

But I would lean on not change it at all.

Reply:
Yes, let me change that :)